lib/FusionInventory/Agent/Tools/Batteries.pm

```diff
@@ -43,6 +45,42 @@ sub sanitizeBatterySerial {
     return hex2dec($serial);
 }
 
+sub getCanonicalVoltage {
+    my ($value) = @_;
+
+    return unless $value;
+
+    # We expect to return voltage in mV
+    if ($value =~ /^(\d+) mV$/i) {
+        return int($1);
+    } elsif ($value =~ /^(\d+) V$/i) {
+        return int($1) * 1000;
+    }
+
+    return $value;
+}
+
+sub getCanonicalCapacity {
+    my ($value, $voltage) = @_;
+
+    return unless $value;
+
+    # We expect to return capacity in mWh, $voltage is expected to be in mV
```

Comment:
Typo _mV_ => _mAh_

Reply:
This is not a typo, I expect voltage in mV unit.